charmcraft/commands/build.py

```diff
@@ -57,6 +57,7 @@
     "LICENSE",
     "icon.svg",
     "README.md",
+    "actions",
```

Comment:
from our conversations last week, this list and CHARM_FILES is the list of things I would not do at the plugin level, but instead as an action of packing.

Reply:
I don't remember that conversation, maybe we were talking about different things? Let's talk again.

That said, maybe we can include this change to fix the problem, and aim for a refactor later?

Reply:
As we discussed in the call, the approach coded here is the correct.